### PR TITLE
BZ1725848 Release Notes

### PR DESCRIPTION
**Pull request by bgaydosrh**
For these release notes, we need a good SME/Dev review, tagging the following people who are mentioned in the bug:
@rhrazdil @nellyc @dankenigsberg @aburdenthehand 

- In particular check the wording of this note: RWX (Read, Write, Execute) file systems are the only supported storage for Live Migration, UI, and importing VMWare virtual machines. 

- Should the above note have a BZ number to associate with it? it originated from QE feedback in this doc: https://docs.google.com/document/d/16LPmRvY8cV9EaIFuAi3HH11xzwKj1kBMvCHAYzUVvW0/edit# [last row]

This is for CNV 2.0.

Thanks,
Bob

### CODE REVIEW

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (dankenigsberg) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: The bug you cite is about a gui-only problem. Fir the urgent issue which breaks masquerade everywhere, I'm aware only of an upstream issue https://github.com/kubevirt/kubevirt/issues/2400

**Review by dankenigsberg — CHANGES_REQUESTED** [review]

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Thanks Dan. I'll swap that link out for the one you supply re this issue and update this PR shortly.

Bob

Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Made change. Not sure if we allow github links in Rel Notes (I have a question out about that). @dankenigsberg let me know what you think about what I have done and I will merge.

Bob

**Comment by bgaydosrh** [comment]
@openshift/team-documentation please peer review.
Label for openshift-enterprise 4.1 and 4.2. For CNV 2.0

**Comment by bgaydosrh** [comment]
QE is questioning whether this warrants an RN (https://bugzilla.redhat.com/show_bug.cgi?id=1722419). Tomas had recommended it but waiting on go-ahead from @vikram-redhat  before we merge.

**Comment by bgaydosrh** [comment]
It looks like we're good to go on this PR. Irina ack'ed that there was no workaround for 1725848. It is assigned to a UI dev now. The other BZ (https://bugzilla.redhat.com/show_bug.cgi?id=1722419) she has verified the content is correct.

Going forward with the merge.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bergerhoffer) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: Just curious if there should there be a BZ link at the end of this, like how there is for all the other known issues.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bergerhoffer) [inline cnv/cnv_release_notes/cnv-release-notes.adoc]: Same question about the BZ link.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Done.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: No BZ link as it is an upstream issue. If you know if github links are permitted I can add one of those.

Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: Actually I can add a BZ link for this blurb. I was confusing it with the other missing link below.

File: cnv/cnv_release_notes/cnv-release-notes.adoc
Comment (bgaydosrh) [reply cnv/cnv_release_notes/cnv-release-notes.adoc]: This Release Note came directly from QE feedback. I will try to find what bug it is associated with.

**Pull request merged**
